Sources/ActivityIndicators/Animations/ActivityIndicatorAnimationRupee.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  ActivityIndicatorAnimationRupee.swift
+//  IBAnimatable
+//
+//  Created by phimage on 07/05/2018.
+//  Copyright © 2018 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+
+public class ActivityIndicatorAnimationRupee: ActivityIndicatorAnimating {
+
+  // MARK: Properties
+
+  fileprivate let duration: CFTimeInterval = 1
```

Comment:
We can use `private` now

Reply:
Could be done later. I just copy paste.
I want to make it public, editable later.